carts/views.py

```diff
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Sum, F
+
+from carts.models    import Cart
+from users.models    import User
+from products.models import Product, Size
+from users.utils     import login_decorator
+
+class CartView(View):
+    @login_decorator
+    def get(self, request):            
+        carts = Cart.objects.filter(user_id = request.user.id)
+        total_price = carts.aggregate(price = Sum(F("count") * F("product__price")))
+
+        response = {
+            "cart" : [{
+            "cart_id" : cart.id,
+            "name"    : cart.product.name,
+            "image"   : cart.product.image_url,
+            "color"   : cart.product.color.name,
+            "size"    : cart.size.name,
+            "count"   : cart.count,
+            "price"   : int(cart.count * cart.product.price)
+        } for cart in carts],
```

Comment:
```suggestion
            "cart" : [{
                "cart_id" : cart.id,
                "name"    : cart.product.name,
                "image"   : cart.product.image_url,
                "color"   : cart.product.color.name,
                "size"    : cart.size.name,
                "count"   : cart.count,
                "price"   : int(cart.count * cart.product.price)
            } for cart in carts],
```
cart라는 key 있는 list 데이터들이 cart랑 같은 레벨에 있는 것처럼 보이지 않게 다음과 같이 수정하면 좋을 것 같아요.

Reply:
cart 안의 데이터들 안쪽으로 들여쓰기하였습니다.